scripts/validate-branch-name.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+local_branch_name="$(git rev-parse --abbrev-ref HEAD)"
+
+valid_branch_regex='^((feature|improvement|bugfix|hotfix|release)\/[a-zA-Z0-9\-]+)$'
```

Comment:
Probably your regex lacks of `.` and `_` I think these two are sometimes used. e.g. `release/1.1.0`

Reply:
Fair point.